main/lsp/protocol.cc

```diff
@@ -140,12 +146,18 @@ optional<unique_ptr<core::GlobalState>> LSPLoop::runLSP(shared_ptr<LSPInput> inp
                     messageQueue.pendingRequests.push_back(move(msg));
                 }
             },
-            [&messageQueue, &messageQueueMutex, logger = logger](int watchmanExitCode) {
+            [&messageQueue, &messageQueueMutex, logger = logger, config = this->config](int watchmanExitCode,
+                                                                                        string const &msg) {
                 {
                     absl::MutexLock lck(&messageQueueMutex);
                     if (!messageQueue.terminate) {
                         messageQueue.terminate = true;
                         messageQueue.errorCode = watchmanExitCode;
+                        if (watchmanExitCode != 0) {
+                            auto params = make_unique<ShowMessageParams>(MessageType(1 /* error */), msg);
```

Comment:
```suggestion
                            auto params = make_unique<ShowMessageParams>(MessageType::Error, msg);
```

you can use the enum name here (and all other places)